Program.cs

```diff
@@ -18,17 +18,20 @@ static void Main()
             return;
         }
 
-        // Console.WriteLine("Enter the operation (+, -, *, /):");
-        // char operation = Console.ReadKey().KeyChar;
+        Console.WriteLine("Enter the operation (+, *):");
+        string operation = Console.ReadLine();
         Console.WriteLine(); // Move to the next line after reading the operation key
-        char operation = '+';
         double result;
         switch (operation)
```

Comment:
@coderabbitai  transform this case statement into a function

Reply:
@coderabbitai add a case for All that would show the result of both addition and multiplication

Reply:
@coderabbitai modify to allow user to input multiple sets of numbers, so they can do 7 + 7, and then be able to do 3 * 7 if they choose to

Reply:
@coderabbit add cases for division and subtraction

Reply:
@coderabbitai add cases for division and subtraction